src/ovp_pipeline/commands/score_sources.py

```
@@ -0,0 +1,188 @@
+    """Stream sources lazily — caller may break out as soon as it has
+    enough (e.g. ``--limit``) without paying for a full FS walk.
+
+    Note: ``rglob`` itself returns paths in arbitrary order; that's
+    fine for the limit case (any N sources is a valid sample).
+    """
+    for f in (vault_dir / "50-Inbox" / "03-Processed").rglob("*.md"):
+        if not f.is_file():
+            continue
+        if since is not None:
+            try:
+                mtime = datetime.fromtimestamp(f.stat().st_mtime)
```

Ruff (0.15.12)

[warning] `datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)